core/src/main/scala/org/apache/spark/deploy/SparkSubmit.scala

```diff
@@ -825,6 +837,43 @@ object SparkSubmit extends CommandLineUtils {
                       .mkString(",")
     if (merged == "") null else merged
   }
+
+  /**
+   * Download a list of remote files to temp local files. If the file is local, the original file
+   * will be returned.
+   * @param fileList A comma separated file list, it cannot be null.
```

Comment:
Nit: no need to add the comment `it cannot be null`. Just add an assert at the beginning of the function to ensure it.